modules/apps/27-interchain-accounts/ibc_module.go

```diff
@@ -1,20 +1,17 @@
 package interchain_accounts
```

Comment:
I think we agreed on slack msgs that we'd rename this to `package ica`, right? 
Doesn't necessarily need to be included here, I can do it as part of the pkg split if you want. Just a mental reminder :)

Reply:
add to the controller/host split issue?

Reply:
Will do!